.github/workflows/integration_tests.yml

```diff
@@ -31,26 +33,45 @@ on:
       ios_version:
         description: 'iOS device version'
         default: '11.4'
+      use_expanded_matrix:
+        description: 'Enabling (1) will run an extended test matrix instead of your config defined above'
```

Comment:
nit: for consistency, keep the same description across all workflows

Reply:
The description here is different to avoid confusion .  This particular workflow has a lot of parameters, almost all of which will be ignored if an expanded matrix is selected.  I didn't just want to say "Use and Expanded Matrix?" here and have the user wonder why they defined to only run a linux datbase api build but suddently the expanded matrix meant that we ignored those configuration changes.  I'll try to tweek it so that at last the begining conforms with the other scripts.

Reply:
Updated, but if you still think that all descriptoins should be the same then please let me know!